tutorials/MD/04_Trajectory_Analysis.ipynb

```diff
@@ -0,0 +1,265 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualise the simulation\n",
+    "\n",
+    "The simplest and easiest type of analysis you should always do is to look at it with your eyes! Your eyes will tell if you something strange is happening immediately. A numerical analysis may not. Let us look at the simulations on VMD.\n",
```

Comment:
Again here I propose we make VMD optional and have nglview as the first option.

here it should just be a case of calling MDAnalysis, creating a universe, getting the protein atomgroup out and then passing that to nglview , as detailed here: http://nglviewer.org/nglview/latest/api.html#nglview.show_mdanalysis

Reply:
NGLView instructions included. VMD reminder that it is optional too.